cpanfile

```diff
@@ -33,6 +33,7 @@ requires 'Mojolicious::Lite';
 requires 'Net::DBus';
 requires 'Net::SNMP';
 requires 'Net::SSH2';
+requires 'Net::Telnet';
```

Comment:
Cherry pick https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/3880f711b9dc1e73ecf77e171d727d3713246d49 and revert that line here